ansible/roles/tilemaker/templates/bicycle/style.json

```diff
@@ -133,6 +133,36 @@
       "layout": {"visibility": "visible"},
       "paint": {"fill-color": "hsla(30, 19%, 90%, 0.4)"}
     },
+    {
+      "id": "landuse-sport",
+      "type": "fill",
+      "metadata": {"mapbox:group": "1444849388993.3071"},
+      "source": "openmaptiles",
+      "source-layer": "landuse",
+      "filter": ["==", "class", "pitch"],
+      "layout": {"visibility": "visible"},
+      "paint": {
+        "fill-color": "#63b063",
+        "fill-opacity": {"base": 1, "stops": [[0, 0.9], [10, 0.3]]}
+      }
+    },
+    {
```

Comment:
If we put the tracks into the landcover layer then we can remove this style definition.